frontend/pkg/frontend/ocm.go

```diff
@@ -128,64 +128,63 @@ func convertNodeDrainTimeoutCSToRP(in *arohcpv1alpha1.Cluster) int32 {
-// 		return api.CustomerManagedEncryptionProfile{
-// 			EncryptionType: api.CustomerManagedEncryptionType(customerManaged.EncryptionType()),
-// 			Kms:            convertKmsEncryptionCSToRP(in),
-// 		}
-// 	}
-// 	return api.CustomerManagedEncryptionProfile{}
-// }
-//
-// func convertKmsEncryptionCSToRP(in *arohcpv1alpha1.Cluster) *api.KmsEncryptionProfile {
-//
-// 	if kms, ok := in.Azure().EtcdEncryption().DataEncryption().CustomerManaged().GetKms(); ok {
-// 		if activeKey, ok := kms.GetActiveKey(); ok {
-// 			return &api.KmsEncryptionProfile{
-// 				ActiveKey: api.KmsKey{
-// 					Name:      activeKey.KeyName(),
-// 					VaultName: activeKey.KeyVaultName(),
-// 					Version:   activeKey.KeyVersion(),
-// 				},
-// 			}
-// 		}
-// 	}
-// 	return nil
-// }
-
-// func convertEtcdRPToCS(in api.EtcdProfile) *arohcpv1alpha1.AzureEtcdEncryptionBuilder {
-// 	azureEtcdDataEncryptionBuilder := arohcpv1alpha1.NewAzureEtcdDataEncryption().KeyManagementMode(convertKeyManagementModeTypeRPToCS(in.DataEncryption.KeyManagementMode))
-// 	if in.DataEncryption.CustomerManaged.Kms != nil || in.DataEncryption.CustomerManaged.EncryptionType != "" {
-// 		azureEtcdDataEncryptionCustomerManagedBuilder := arohcpv1alpha1.NewAzureEtcdDataEncryptionCustomerManaged().EncryptionType(string(in.DataEncryption.CustomerManaged.EncryptionType))
-// 		azureKmsKeyBuilder := arohcpv1alpha1.NewAzureKmsKey().KeyName(in.DataEncryption.CustomerManaged.Kms.ActiveKey.Name).KeyVaultName(in.DataEncryption.CustomerManaged.Kms.ActiveKey.VaultName).KeyVersion(in.DataEncryption.CustomerManaged.Kms.ActiveKey.Version)
-// 		azureKmsEncryptionBuilder := arohcpv1alpha1.NewAzureKmsEncryption().ActiveKey(azureKmsKeyBuilder)
-// 		azureEtcdDataEncryptionCustomerManagedBuilder = azureEtcdDataEncryptionCustomerManagedBuilder.Kms(azureKmsEncryptionBuilder)
-// 		azureEtcdDataEncryptionBuilder.CustomerManaged(azureEtcdDataEncryptionCustomerManagedBuilder)
-// 	}
-// 	return arohcpv1alpha1.NewAzureEtcdEncryption().DataEncryption(azureEtcdDataEncryptionBuilder)
-
-// }
+func convertKeyManagementModeTypeCSToRP(keyManagementModeCS string) (keyManagementModeRP api.EtcdDataEncryptionKeyManagementModeType) {
+	switch keyManagementModeCS {
+	case "platform_managed":
+		keyManagementModeRP = api.EtcdDataEncryptionKeyManagementModeTypePlatformManaged
+	case "customer_managed":
+		keyManagementModeRP = api.EtcdDataEncryptionKeyManagementModeTypeCustomerManaged
+	}
+	return
+}
+
+func convertKeyManagementModeTypeRPToCS(keyManagementModeRP api.EtcdDataEncryptionKeyManagementModeType) (keyManagementModeCS string) {
+	switch keyManagementModeRP {
+	case api.EtcdDataEncryptionKeyManagementModeTypePlatformManaged:
+		keyManagementModeCS = "platform_managed"
+	case api.EtcdDataEncryptionKeyManagementModeTypeCustomerManaged:
+		keyManagementModeCS = "customer_managed"
+	}
+	return
+}
+func convertCustomerManagedEncryptionCSToRP(in *arohcpv1alpha1.AzureEtcdDataEncryption) *api.CustomerManagedEncryptionProfile {
```

Comment:
nit: Missing empty line here.
```suggestion

func convertCustomerManagedEncryptionCSToRP(in *arohcpv1alpha1.AzureEtcdDataEncryption) *api.CustomerManagedEncryptionProfile {
```